tests/rptest/chaos_tests/single_fault_test.py

```diff
@@ -101,6 +111,25 @@ def _do_run(self, workload: workloads.WorkloadServiceBase,
                 for node in workload.nodes:
                     before_heal_info[node.name] = workload.info(node)
 
+                if check_progress_during_fault is not None:
+                    results = []
+                    for node_name in before_heal_info.keys():
+                        delta = before_heal_info[
+                            node_name].succeeded_ops - after_fault_info[
+                                node_name].succeeded_ops
```

Comment:
probably out of scope, but I think we should improve our python code formatting automation

Reply:
added brackets so that it looks nicer :)